script/licenses/update-license-dump.ts

```diff
@@ -0,0 +1,60 @@
+import * as path from 'path'
+import * as fs from 'fs-extra'
+import { promisify } from 'util'
+
+import { licenseOverrides } from './license-overrides'
+
+import * as legalEagle from 'legal-eagle'
+import { getVersion } from '../../app/package-info'
+
+export async function updateLicenseDump(
+  projectRoot: string,
+  outRoot: string
+): Promise<void> {
+  const appRoot = path.join(projectRoot, 'app')
+  const outPath = path.join(outRoot, 'static', 'licenses.json')
+
+  let summary = await promisify(legalEagle)({
```

Comment:
Since you use this multiple times, it might be clearer to do this:

```ts
import * as legalEagle_ from 'legal-eagle'
const legalEagle = promisify(legalEagle)
```

(I would’ve mentioned this at first but I didn’t see the second usage 📱)